app/preload/preload.js

```diff
@@ -8,6 +8,6 @@ window.jitsiNodeAPI = {
     createElectronStorage,
     osUserInfo: os.userInfo,
     shellOpenExternal: shell.openExternal,
-    jitsiMeetElectronUtils
+    jitsiMeetElectronUtils,
+    ipcRenderer
```

Comment:
@andrei-gavrilescu Can you PTAL? I wonder if it's safe to do this.

Reply:
Hello everyone.
The electron recommendation in this case is to expose a custom API rather than directly expose node functionality to a renderer that loads remote content. 
Wrapping the ipcRenderer logic inside a function and exposing that would be the preferred way.
See: https://www.electronjs.org/docs/tutorial/security point 2.

Reply:
That's what I was fearing. Thanks a lot @andrei-gavrilescu ! @kempu can you please adjust the code?

Reply:
Thanks! Will try to take a look at it as soon as I get some extra time.

Reply:
Thanks for catching that. Hope the new approach is better :)